core/modules/node/node.pages.inc

```diff
@@ -330,7 +330,7 @@ function node_form($form, &$form_state, Node $node) {
     $form['options']['status'][NODE_SCHEDULED]['#access'] = FALSE;
   }
 
-  if ($node_type->settings['scheduling_enabled'] && user_access('administer nodes')) {
+  if ($node_type->settings['scheduling_enabled']) {
```

Comment:
This IF statement is now the same condition as the one a few lines earlier (below `Adjustments to the status radios if scheduling is enabled/disabled.`). We should consolidate these into a single IF/ELSE.

Reply:
Fixed in https://github.com/backdrop/backdrop/pull/2683